Importing @keyv/redis causes TypeScript error: Cannot find name 'Value'.

**Describe the bug**
Simply loading @keyv/redis in a TS project causes transpilation error:
```
> tsc

node_modules/@keyv/redis/src/index.d.ts:6:63 - error TS2304: Cannot find name 'Value'.

6 declare class KeyvRedis extends EventEmitter implements Store<Value> {
                                                                ~~~~~

node_modules/@keyv/redis/src/index.d.ts:13:28 - error TS2304: Cannot find name 'Value'.

13  get(key: string): Promise<Value>;
                              ~~~~~

node_modules/@keyv/redis/src/index.d.ts:16:22 - error TS2304: Cannot find name 'Value'.

16  ): Array<StoredData<Value>> | Promise<Array<StoredData<Value>>> | undefined;
                        ~~~~~

node_modules/@keyv/redis/src/index.d.ts:16:57 - error TS2304: Cannot find name 'Value'.

16  ): Array<StoredData<Value>> | Promise<Array<StoredData<Value>>> | undefined;
                                                           ~~~~~

node_modules/@keyv/redis/src/index.d.ts:17:26 - error TS2304: Cannot find name 'Value'.

17  set(key: string, value: Value, ttl?: number): any;
                            ~~~~~


Found 5 errors in the same file, starting at: node_modules/@keyv/redis/src/index.d.ts:6
```

**How To Reproduce (best to provide workable code or tests!)**
Im upgrading from
* "@keyv/redis": "2.3.7",
* "keyv": "4.3.2"
but experience the above error when building with TypeScript 4.8.4

Upgrading to
* "@keyv/redis": "2.5.1",
* "keyv": "4.5.0"

the following examples causes the above compilation errors.

`RedisCache.ts` - Note that this is a dumbed down snippet from my project.
```typescript
import Keyv from 'keyv';
import KeyvRedis from '@keyv/redis';

import { ICache } from './ICache';

export type RedisCacheOptions = {
	namespace: string,
	ttl: number
};

export class RedisCache<T> implements ICache<T> {
	private readonly keyv: Keyv;

	constructor(options: RedisCacheOptions) {
		this.keyv = new Keyv({
			namespace: options.namespace,
			ttl: options.ttl
		});
	}

	public async clear(): Promise<void> {
		return this.keyv.clear();
	}

	public async delete(key: string): Promise<void> {
		await this.keyv.delete(key);

		return undefined;
	}

	public async get(key: string): Promise<any | undefined> {
		return this.keyv.get(key);
	}

	public async set(key: string, value: any, ttl?: number): Promise<void> {
		await this.keyv.set(key, value, ttl);

		return undefined;
	}
}
```

Looking at the content of https://github.com/jaredwray/keyv/blob/f060943bd21b4de17b5b7be956debad86d6633d1/packages/redis/src/index.d.ts I don't see where `Value` should come from. Also, none of the other apaters' typedefinitions forward or define this themselves. It's only defined as `any` in the typedefinitions of `keyv` itself.
Should the adapters themselves not let this be customisable? Something like
```typescript
declare class KeyvRedis<Value = any> extends EventEmitter implements Store<Value> {}
```

--- a/packages/etcd/src/index.d.ts
+++ b/packages/etcd/src/index.d.ts
@@ -2,7 +2,7 @@ import {EventEmitter} from 'events';
 import type Etcd3 from 'etcd3';
 import type {Store, StoredData} from 'keyv';
 
-declare class KeyvEtcd extends EventEmitter implements Store<Value> {
+declare class KeyvEtcd<Value=any> extends EventEmitter implements Store<Value> {
 	ttlSupport: any;
 	opts: any;
 	client: Etcd3;

--- a/packages/memcache/src/index.d.ts
+++ b/packages/memcache/src/index.d.ts
@@ -2,7 +2,7 @@
 import {EventEmitter} from 'node:events';
 import {Store, StoredData} from 'keyv';
 
-declare class KeyvMemcache extends EventEmitter implements Store<Value> {
+declare class KeyvMemcache<Value=any> extends EventEmitter implements Store<Value> {
 	ttlSupport: boolean;
 	namespace?: string | undefined;
 

--- a/packages/mongo/src/index.d.ts
+++ b/packages/mongo/src/index.d.ts
@@ -2,7 +2,7 @@ import {EventEmitter} from 'events';
 import GridFSBucket from 'mongodb';
 import {Store, StoredData} from 'keyv';
 
-declare class KeyvMongo extends EventEmitter implements Store<Value> {
+declare class KeyvMongo<Value=any> extends EventEmitter implements Store<Value> {
 	readonly ttlSupport: false;
 	opts: Record<string, any>;
 	connect: Promise<any>;

--- a/packages/mysql/src/index.d.ts
+++ b/packages/mysql/src/index.d.ts
@@ -3,7 +3,7 @@ import {EventEmitter} from 'events';
 import type {Store, StoredData} from 'keyv';
 
 export = KeyvMysql;
-declare class KeyvMysql extends EventEmitter implements Store<Value> {
+declare class KeyvMysql<Value=any> extends EventEmitter implements Store<Value> {
 	readonly ttlSupport: false;
 	opts: any;
 	query: (sqlString: any) => any;

--- a/packages/offline/src/index.d.ts
+++ b/packages/offline/src/index.d.ts
@@ -1,7 +1,7 @@
 import EventEmitter from 'node:events';
 import type {Store, StoredData} from 'keyv';
 
-declare class KeyvOffline extends EventEmitter implements Store<Value> {
+declare class KeyvOffline<Value=any> extends EventEmitter implements Store<Value> {
 	proxy: any;
 	constructor(keyv: any);
 	get(key: string): Promise<Value>;

--- a/packages/postgres/src/index.d.ts
+++ b/packages/postgres/src/index.d.ts
@@ -3,7 +3,7 @@ import {EventEmitter} from 'events';
 import type {Store, StoredData} from 'keyv';
 
 export = KeyvPostgres;
-declare class KeyvPostgres extends EventEmitter implements Store<Value> {
+declare class KeyvPostgres<Value=any> extends EventEmitter implements Store<Value> {
 	readonly ttlSupport: false;
 	opts: any;
 	query: (sqlString: any) => any;

--- a/packages/redis/src/index.d.ts
+++ b/packages/redis/src/index.d.ts
@@ -3,7 +3,7 @@ import {EventEmitter} from 'events';
 import type {Store, StoredData} from 'keyv';
 import type {Redis, Cluster} from 'ioredis';
 
-declare class KeyvRedis extends EventEmitter implements Store<Value> {
+declare class KeyvRedis<Value=any> extends EventEmitter implements Store<Value> {
 	readonly ttlSupport: false;
 	namespace?: string | undefined;
 	opts: Record<string, unknown>;

--- a/packages/sqlite/src/index.d.ts
+++ b/packages/sqlite/src/index.d.ts
@@ -4,7 +4,7 @@ import type Database from 'better-sqlite3';
 import type {Store, StoredData} from 'keyv';
 
 export = KeyvSqlite;
-declare class KeyvSqlite extends EventEmitter implements Store<Value> {
+declare class KeyvSqlite<Value=any> extends EventEmitter implements Store<Value> {
 	readonly ttlSupport: false;
 	namespace?: string | undefined;
 	opts: any;

--- a/packages/tiered/src/index.d.ts
+++ b/packages/tiered/src/index.d.ts
@@ -2,7 +2,7 @@ import {EventEmitter} from 'events';
 import type {Store, StoredData} from 'keyv';
 
 export = KeyvTiered;
-declare class KeyvTiered extends EventEmitter implements Store<Value> {
+declare class KeyvTiered<Value=any> extends EventEmitter implements Store<Value> {
 	constructor(options: KeyvTiered.Options);
 	get(key: string): Promise<Value>;
 	getMany?(

